openhtf/core/test_record.py

```diff
@@ -71,16 +71,15 @@ def add_outcome_details(self, code, description=''):
 class PhaseRecord(  # pylint: disable=no-init
     mutablerecords.Record(
         'PhaseRecord', ['name', 'codeinfo'],
-        {'measurements': None,
-         'start_time_millis': None, 'end_time_millis': None,
-         'attachments': dict, 'result': None})):
+        {'start_time_millis': None, 'end_time_millis': None,
+         'measurements': dict, 'attachments': dict, 'result': None})):
```

Comment:
This is documenting the type, really. This becomes a dict after the phase has executed, which is right around the time this record is created anyway, and nobody checks the measurements for being `None` (or at all). Rather, I was confused as to the type of measurements (is it a list here like it is elsewhere?) and wanted to document that